src/Core/FlatpakBackend.vala

```diff
@@ -72,6 +74,12 @@ public class AppCenterCore.FlatpakBackend : Backend, Object {
         appstream_pool = new AppStream.Pool ();
         appstream_pool.set_cache_flags (AppStream.CacheFlags.NONE);
         package_list = new Gee.HashMap<string, Package> (null, null);
+        monitor = installation.create_monitor ();
+        monitor.changed.connect (() => {
+            debug ("Flatpak installation changed.");
+            // Flatpak cache update tasks go here?
+            Client.get_default ().update_cache.begin (true);
```

Comment:
Better to just call `refresh_cache` on the current instance of `FlatpakBackend` rather than the whole client. Or else we end up updating the caches for the PackageKit backend too.